specs/commandline/sign.md

```diff
@@ -27,58 +41,79 @@ Flags:
+# - User creates keys and certificates in a 3rd party key provider (e.g. key vault, key management service). The signing plugin installed in previous step must support generating signatures using this key provider.
 
-# Sign a container image using the default signing key
-notation sign <registry>/<repository>:<tag>
+# Add a default signing key referencing the key identifier for the remote key, and the plugin associated with it.
+notation key add --default --name <key_name> --plugin <plugin_name> --id <remote_key_id>
 
-# Or using container image digests instead of tags. A container image digest uniquely and immutably identifies a container image.
+# sign an artifact stored in a registry using a remote key
 notation sign <registry>/<repository>@<digest>
 ```
 
-### Sign a container image using a remote key
+An example for a successful signing:
 
 ```shell
-# Prerequisites: 
-# - A compliant signing plugin is installed in notation. See notation plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
-# - User creates keys and certificates in a 3rd party key provider (e.g. key vault, key management service). The signing plugin installed in previous step must support generating signatures using this key provider.
+$ notation sign localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Signing of artifact <reference> succeeded and the signature is attached to localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9.
+```
 
-# Add a default signing key referencing the key identifier for the remote key, and the plugin associated with it.
-notation key add --default --name <key_name> --plugin <plugin_name> --id <remote_key_id>
+### Sign an OCI artifact using COSE signature format
+
+```shell
+# Prerequisites: 
+# A default signing key is configured using CLI "notation key"
 
-# sign a container image using a remote key
-notation sign <registry>/<repository>:<tag>
+# Use option "--signature-format" to set the signature format to COSE.
+$ notation sign --signature-format cose <registry>/<repository>@<digest>
 ```
 
-### Sign an OCI artifact using the default signing key
+### Sign an OCI artifact stored in a registry using the default signing key
 
 ```shell
 # Prerequisites: 
 # A default signing key is configured using CLI "notation key"
 
 # Use a digest that uniquely and immutably identifies an OCI artifact.
-notation sign <registry>/<repository>@<digest>
+$ notation sign <registry>/<repository>@<digest>
 ```
 
-### Sign a container image and specify the signature expiry duration, for example 1 day
+### Sign an OCI artifact stored in a registry and specify the signature expiry duration, for example 24 hours
 
 ```shell
-notation sign --expiry 1d <registry>/<repository>:<tag>
+notation sign --expiry 24h <registry>/<repository>@<digest>
```

Comment:
IMO day makes more sense because user will rarely want an signature to expire in less than a day.

Reply:
Currently `day` is not support yet.

Reply:
Opened issue https://github.com/notaryproject/notation/issues/451 and tagged it for rc2

Reply:
Thanks Pritesh. There was an issue https://github.com/notaryproject/notation/issues/423 I created earlier.